pipelines/rj_smas__disparo_pic_lembrete/schemas.py

```
@@ -0,0 +1,165 @@
+
+        Args:
+            v: String do telefone a ser validada
+
+        Returns:
+            String do telefone validada
+
+        Raises:
+            ValueError: Se não for string ou estiver vazio
+        """
+        if not isinstance(v, str):
+            raise ValueError("Telefone deve ser uma string")
```

Ruff (0.14.1)

Prefer `TypeError` exception for invalid type

(TRY004)